src/commands/lambda/constants.ts

```diff
@@ -165,3 +164,7 @@ export const FRAMEWORK_FILES_MAPPING = new Map([
 ])
 
 export const LAMBDA_PROJECT_FILES = [...FLARE_PROJECT_FILES, ...FRAMEWORK_FILES_MAPPING.keys()]
+
+// Configures max number of attempts and exponential backoff function for AWS requests
+// First retry is attempt 1
+export const RETRY_STRATEGY = new ConfiguredRetryStrategy(4, (attempt: number) => 1000 * 2 ** (attempt - 1))
```

Comment:
I'd probably rename `RETRY_STRATEGY` to something more descriptive. If we have two different retry strategies, the name `RETRY_STRATEGY` wouldn't make much sense.

Reply:
I renamed it to `EXPONENTIAL_BACKOFF_RETRY_STRATEGY`. Let me know if that's too verbose